source/reference/beta-program.txt

```diff
@@ -97,6 +97,11 @@ Beta Features
        guidelines and restrictions for migrating between older server 
        versions, you must have access to the {+c2c-beta-program-short+} binary.
 
+.. note:: 
```

Comment:
[question] Do you think we should add a note on the [reverse ](https://deploy-preview-424--docs-cluster-to-cluster-sync.netlify.app/reference/api/reverse/#std-label-c2c-api-reverse)page?

Reply:
@ltran-mdb2 there is a section called 'limitations' and the ticket mentioned that the incompatibility is not mentioned anywhere else, so I think that would be wise. Would it be best for me to do it in this ticket, or open up a new one?

Reply:
I think you can do it in this ticket!

Reply:
@ltran-mdb2 added the link!